src/component/link/style/tool/_size.scss

```diff
@@ -24,8 +24,9 @@
 
   $space-y: max($line-height, $max-icon-height);
   $padding-x: space(map_get($size-settings, padding-x));
+  $padding-y: if($min-height - $space-y > 0, ($min-height - $space-y) / 2, 0);
```

Comment:
il me semble que t'as min comme fonction math que tu peux utiliser là
https://sass-lang.com/documentation/values/calculations#min-and-max